_posts/2022-07-19-react-to-full-stack.md

```diff
@@ -0,0 +1,606 @@
+---
+layout: post
+title: React To Full Stack
+date: 2022-07-19
+tags: [react, mui, typescript, aws]
```

Comment:
i think you should replace `typescript` with `javascript` since there is no TS in this blog.

Reply:
Agreed & applied